test/onnx/test_pytorch_onnx_onnxruntime.py

```diff
@@ -2642,6 +2642,36 @@ def forward(self, *tensor_list):
         x = torch.randn(3, 4)
         self.run_test(EinsumModelTranspose(), input=(x,))
 
+    @unittest.skip("Enable this once ORT version is updated")
```

Comment:
could you add a test in test_operators.py as well to make sure that broadcast_tensor is being exported correctly